dotnet-desktop-guide/framework/wpf/advanced/how-to-use-special-characters-in-xaml.md

```diff
@@ -11,21 +11,23 @@ helpviewer_keywords:
 ms.assetid: a57776d1-f353-4794-afa0-bfa3c712ed1c
 ---
 # How to: Use Special Characters in XAML
-Markup files that are created in Visual Studio are automatically saved in the Unicode UTF-8 file format, which means that most special characters, such as accent marks, are encoded correctly. However, there is a set of commonly-used special characters that are handled differently. These special characters follow the World Wide Web Consortium (W3C) XML standard for encoding.  
-  
- The following table shows the syntax for encoding this set of special characters:  
-  
-|Character|Syntax|Description|  
-|---------------|------------|-----------------|  
-|<|`&lt;`|Less than symbol.|  
-|>|`&gt;`|Greater than sign.|  
-|&|`&amp;`|Ampersand symbol.|  
-|"|`&quot;`|Double quote symbol.|  
-  
+Markup files that are created in Visual Studio are automatically saved in the Unicode UTF-8 file format, which means that most special characters, such as accent marks, are encoded correctly. However, there is a set of commonly-used special characters that are handled differently. These special characters follow the World [Wide Web Consortium (W3C) XML standard for encoding](https://www.w3resource.com/xml/reserved-markup-characters.php).
+
+The following table shows the syntax for encoding this set of special characters:
+
+| Character | Syntax   | Description          |  |
+|-----------|----------|----------------------|--|
+| <         | `&lt;`   | Less than symbol.    |  |
+| >         | `&gt;`   | Greater than sign.   |  |
+| &         | `&amp;`  | Ampersand symbol.    |  |
+| "         | `&quot;` | Double quote symbol. |  |
+| '         | `&apos;` | Single quote symbol. |  |
+
```

Comment:
Changes were between here, rest of file was killing off the doublespaces at ends of lines.